internal/hud/renderer.go

```diff
@@ -188,10 +188,8 @@ func keyLegend(v view.View, vs view.ViewState) string {
 	return defaultKeys
 }
 
-func isInError(res view.Resource) bool {
-	return res.LastBuild().Error != nil ||
-		(res.ResourceInfo != nil && statusColors[res.ResourceInfo.Status()] == cBad) ||
-		isCrashing(res)
+func isInError(res view.Resource, triggerMode model.TriggerMode) bool {
```

Comment:
it uh. seems somewhat questionable to made this decision by status color and not the other way around? but also eh, it's no worse than it was before 😅

Reply:
yeah, really we should have an enum for bad/pending/good, make `statusColor` return that enum, and delete `isInError`

maybe next time